src/OrbitQt/orbitmainwindow.h

```diff
@@ -56,7 +56,8 @@ class OrbitMainWindow : public QMainWindow {
   static constexpr int kEndSessionReturnCode = 1;
 
   explicit OrbitMainWindow(orbit_qt::TargetConfiguration target_configuration,
-                           orbit_metrics_uploader::MetricsUploader* metrics_uploader = nullptr);
+                           orbit_metrics_uploader::MetricsUploader* metrics_uploader = nullptr,
+                           std::vector<std::string> command_line_args = {});
```

Comment:
Does this need to be a copy of the vector, or would `const std::vector<std::string>&` do as well?

Reply:
Thanks for pointing out this mistake. I didn't notice here. Changed to use `const std::vector<std::string>&`.